README.md

```diff
@@ -162,20 +178,22 @@ See: [Cached State in render](#cached-state-in-render) anti-pattern
 
 Name compound state methods with the `is`, `has` or `can` prefix.
 
-```coffeescript
-# bad
-happyAndKnowsIt: ->
-  @state.happy and @state.knowsIt
+```javascript
+happyAndKnowsIt() {
+  return this.state.happy && this.state.knowsIt;
+},
 
-# good
-isWillingSongParticipant: ->
-  @state.happy and @state.knowsIt
+isWillingSongParticipant() {
```

Comment:
Similar to above, do you want to keep the `good` comment here to be specific and explicit?